src/api/index.ts

```diff
@@ -0,0 +1,43 @@
+import { createClient } from '../util/grpc'
+import {
+  API,
+  EventCreateInputs, EventCreateOutputs, EventStreamInputs,
+  ExecutionCreateInputs, ExecutionGetInputs, ExecutionUpdateInputs, ExecutionCreateOutputs, ExecutionGetOutputs, ExecutionUpdateOutputs, ExecutionStreamInputs,
+  InstanceCreateInputs, InstanceGetInputs, InstanceListInputs, InstanceDeleteInputs, InstanceCreateOutputs, InstanceGetOutputs, InstanceListOutputs, InstanceDeleteOutputs,
+  ServiceCreateInputs, ServiceGetInputs, ServiceListInputs, ServiceDeleteInputs, ServiceCreateOutputs, ServiceGetOutputs, ServiceListOutputs, ServiceDeleteOutputs
+} from './types'
+
+const promisify = (client, method) => request => new Promise((resolve, reject) => client[method](request, (err, res) => err ? reject(err) : resolve(res)))
+
+export default (endpoint: string): API => {
+  const event = createClient('Event', 'protobuf/api/event.proto', endpoint)
+  const execution = createClient('Execution', 'protobuf/api/execution.proto', endpoint)
+  const instance = createClient('Instance', 'protobuf/api/instance.proto', endpoint)
+  const service = createClient('Service', 'protobuf/api/service.proto', endpoint)
+  return {
+    event: {
+      Create: promisify(event, 'Create') as (request: EventCreateInputs) => EventCreateOutputs,
```

Comment:
why `event` in lowercase but `Create` in uppercase?

Reply:
Just wanted to stay close to the grpc definition but I can change that

Reply:
fixed everything in lower case